### ISSUE TITLE
Check/Sanitize PWA install path

### ISSUE THREAD

**Issue by lpointet** [uncatcrea/wp-appkit#300]
For example, we don't want to install the PWA under `wp-admin` directory or outside the current website.

**Comment by lpointet** [comment]
@mleroi @blupu 
Some feedbacks are very welcome here, as I'm not sure everything is under cover:
 - I intentionally let `wp-content` accessible for a PWA installation, open to discussion
 - I tried to avoid installations outside of WordPress itself, but I might have forgotten some configurations?

One thing to mention is that I checked 2 different configurations of WP install: a "normal" single site one, and another single site with WordPress core in a subfolder from the website's root. The second configuration means site URL (where WP is) is different from home URL (where the root is).
Both are working fine, as we're installing PWA in site URL, so it's always where WP resides.

Multisite install could have different behaviours though, since all websites share the same server path, but not the URLs. Didn't check that for the moment.
